### DISCUSSION TITLE
How to code nlmixr2 model when using log transformed concentrations

### DISCUSSION THREAD

**Discussion by jiankelius** [nlmixr2/nlmixr2#286] [Q&A]
Hello, 
I am new with nlmixr2. I would like to know how to code nlmixr2 model when using log transformed concentrations. I am guessing the following: 

```r
cp = center/v
logcp = log(cp)
logcp ~ add(add.err)
```

appreciate your help. 
Thanks,
Jack

**Comment by mattfidler** [comment]
Hi @jiankelius 

This can be used, this is what is used in NONMEM.  In this case:
- Your `DV` must be transformed
- You have to manually transform the values back.
- The objective functions cannot be compared to a simple additive, proportional error (etc) since the data are different.

While it can be used I suggest using this instead:

```r
cp = center/v
cp ~ lnorm(lnorm.sd)
```

When using this method:
- Your `DV` can be on the regular scale
- The simulations generated from the model will always be positive with `cp`
- The objective functions are comparible between additive/proportional and `lnorm`

This is based on dynamic transformation of both sides:

[10.1007/s10928-015-9460-y](https://doi.org/10.1007/s10928-015-9460-y)
